Composer/packages/server/src/controllers/publisher.ts

```diff
@@ -68,9 +68,7 @@ export const PublishController = {
     const profile = profiles.length ? profiles[0] : undefined;
     const extensionName = profile ? profile.type : ''; // get the publish plugin key
 
-    // get token from header
-    const accessToken = req.headers.authorization?.substring('Bearer '.length);
-    log('access token get from header: %s', accessToken);
+    log('access token retrieved from body: %s', accessToken || 'no token provided');
```

Comment:
Is this wise to log the access token?

Reply:
We do it in the auth service as well. Since you have to enable logs via the `DEBUG=composer*` environment variable it seems ok to me. But we can remove it if you think otherwise.